docs/guides/tools/proposer-based-timestamps-runbook.md

```diff
@@ -237,15 +239,22 @@ If the `consensus_quorum_prevote_delay` value approaches the sum of `precision`
 then the value selected for these parameters is too small. Your application will
 need to be modified to update the `SynchronyParams` to have larger values.
 
+Note that the `message_delay` adopted by CometBFT 
+[increases over rounds](../../explanation/core/proposer-based-timestamps.md#adaptive-messagedelay),
+so that the chain does not block forever when it is set to an improper value.
+However, if the standard `message_delay`, used in round 0, is too small, there
+is an important performance impact, and the value of this parameter should be
+updated in order to be aligned with actual message delays in the network.
+
 ### Updating SynchronyParams
 
-The `SynchronyParams` are `ConsensusParameters` which means they are set and updated
+The `SynchronyParams` are Consensus Parameters, which means they are the same for all nodes in the network and are set and updated
 by the application running alongside CometBFT. Updates to these parameters must
 be passed to the application during the `FinalizeBlock` ABCI method call.
 
 If the application was built using the CosmosSDK, then these parameters can be updated
-programmatically using a governance proposal. For more information, see the [CosmosSDK
-documentation](https://hub.cosmos.network/main/governance/submitting.html#sending-the-transaction-that-submits-your-governance-proposal).
+programmatically using a governance proposal. For more information, see the
+[CosmosSDK documentation](https://docs.cosmos.network/v0.45/ibc/proposals.html).
```

Comment:
Note that PTBS is a CometBFT v1 feature, which should be paired with SDK >= 0.50

Reply:
Yes, but there is no link for 0.50 in their doc yet.

Reply:
https://docs.cosmos.network/v0.50/build/modules/gov#proposal-submission